modules/ROOT/pages/release-notes.adoc

```diff
@@ -3,55 +3,54 @@
 ====
 
-
-== New values added
+== New added values
 
 === Bonita now embeds the bpmn-visualization library to improve process monitoring for administrators and users 
 
-Bonitasoft has launched the Process Analytics initiative two years ago, to create a new, Open Source means to monitor process execution using the bpmn diagram generated by Bonita Studio. +
+Bonitasoft has launched the open source project Process Analytics two years ago, as a response to market's need in bpmn process monitoring execution using bpmn standards. +
 The deliverable of this project is called the _bpmn-visualization library_ . You can find a nice array of examples of what it can do in its https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/v0.25.0/examples/index.html[demo web site], and you can even contribute to this initiative in its https://github.com/process-analytics/bpmn-visualization-js[Github repository]. +
-On the other hand, Bonita has been using another library to display process execution data for administrators at both process and case levels. This library had some limitations and was not easy to improve with new features. +
-So it was a very natural choice to eventually integrate the bpmn-visualization library to Bonita, and team up with the Process Analytics team. +
+Till Bonita 2022.2, another library has been used to display process execution data for administrators at both process and case levels. This library had some limitations and was not easy to improve with new features. +
+Hence, integrating the bpmn-visualization library to Bonita, was a very natural choice. +
 
-Some new values are available, starting with the Efficiency Edition of Bonita:
+The new values are available for Bonita Subscription Editions starting with the Efficiency.
   
-  * The users of the Bonita Administration application will see a faster and more robust view of the process execution data (thanks to the library and new REST APIs)
-  * As the bpmn-visualization library improves, we will add more interactivity and more information to provide a better monitoring capacity
-  * Since the pages have been recreated with the UI Designer, developers can:
+  * Bonita Administration application users will get a faster and more robust view on the process execution data (thanks to the library and new REST APIs)
+  * As the bpmn-visualization library evolves, more interactivity and information for a better monitoring capacity will be provided
+  * Since the pages have been created by using Bonita UI Designer, developers can:
         
         ** Use the custom widget embedded in the new pages in their own application pages. The widget has been wrapped into a fragment that makes it very easy to use.
-        ** Offer process execution data visualization to users of their applications, by adjusting the permissions accordingly
+        ** Offer process execution data visualization to specific users by adjusting the permissions accordingly
```

Comment:
What they can do is to add the page directly in their app as and link them. Why to refer to permissions here?

Reply:
Not sure why. This line was already here. I can remove it

Reply:
Jerome suggested a more appropriate communication.